lib/src/local_backend_model.thrift

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+struct NormalFile {
+  1: required binary id,
+  2: required bool executable,
+}
+
+union TreeValue {
+  1: optional NormalFile normal_file,
+  2: optional binary symlink_id,
+  3: optional binary tree_id,
+  4: optional binary conflict_id,
```

Comment:
I think `optional` can be omitted in unions as implied. Not sure what thrift norms are, though.

Reply:
I don't know either, but I agree with removing it, and I skipped it in `op_store_model.thrift`.